hackathon_site/event/jinja2/event/form_base.html

```diff
@@ -24,10 +24,13 @@
         <div class="loginDiv z-depth-3 row">
             <div class="row col s12 flexColCenter {% if self.form_img() %}l7 m8{% endif %}">
                 <h1 class="formH1">{% block form_title %}{% endblock %}</h1>
+                <p>{% block form_description %}{% endblock %}</p>
```

Comment:
Could we move the `<p>` tags inside the block when it's defined? It would be nice to be able to put a h2 or something in here too, or apply different classes (I could use this in my PR)